config/action.d/badips.py

```diff
@@ -80,13 +82,14 @@ class BadIPsAction(ActionBase):
 	"""
 
 	_badips = "http://www.badips.com"
-	_Request = partial(
-		Request, headers={'User-Agent': "Fail2Ban %s" % f2bVersion})
+	def _Request(self, url, **argv):
+		return Request(url, headers={'User-Agent': self.agent}, **argv)
```

Comment:
Could you elaborate a bit more why partial want sufficient? 
Also if indentation is right (of just effect of tabs?) this def is just within __init__